spec/models/service_order_spec.rb

```diff
@@ -33,4 +33,25 @@
       end
     end
   end
+
+  it "should return a short_his_name" do
+    short_name = ServiceOrder.new(his_name: "John Smith")
```

Comment:
``short_name`` isn't a good name, right?

**Cause we create a new instance of ServiceOrder**, so the name should be ```service_order```